doc/classes/CollisionPolygon3D.xml

```diff
@@ -21,7 +21,7 @@
 		</member>
 		<member name="polygon" type="PackedVector2Array" setter="set_polygon" getter="get_polygon" default="PackedVector2Array()">
 			Array of vertices which define the 2D polygon in the local XY plane.
-			[b]Note:[/b] The returned value is a copy of the original. Methods which mutate the size or properties of the return value will not impact the original polygon. To change properties of the polygon, assign it to a temporary variable and make changes before reassigning the class property.
+			[b]Note:[/b] The returned value is a copy of the original. Methods which mutate the size or properties of the return value will not impact the original array. To change properties of the array, assign it to a temporary variable and make changes before reassigning the class property.
```

Comment:
Updated the message to not be specific, I believe this makes each copy directly translatable